pysteps/datasets.py

```diff
@@ -149,6 +158,71 @@ def end(message="Download complete"):
         sys.stdout.write("\n" + message + "\n")
 
 
+def download_mrms_data(dir_path=None, frames=35):
```

Comment:
The example data for MRMS are downloaded from an external source instead of having them stored in pysteps-data like the other datasets. This is perfectly fine, we just need to be sure that this source is active. It seems to me that you're already testing it in `test_datasets.py`, is that right?

Reply:
Yes. However, the archive where the data is stored warns that:

>The IEM is a volunteer effort and receives no funds for facilities or staff from Iowa State University or the State of Iowa. Users of the IEM must therefore recognize that the IEM may be discontinued at any time with little or no notice.
https://mesonet.agron.iastate.edu/disclaimer.php

Reply:
Alright, thanks for the info. We just know that when that happens, the test_datasets will fail and we'll probably need to deprecate that method. 
But if it's only about having a small example set, then perhaps having it in our pysteps-data repo is not a bad idea. What do you think?

Reply:
Good point, it is only 30Mb. I will move them to the pysteps-data.

Reply:
Ok, but I wouldn't just remove your original function! Can we leave it as additional method in the dataset module? Maybe you could just extend it to accept an arbitrary timestamp and number of frames. Although maybe this would go a bit behind the scope of module...

Reply:
I was too drastic :laughing:. With minor refactoring, `download_mrms_data` could be used to retrieve data from the server between two dates. I already have the code snippet needed for that. I would only add a warning in case they try to retrieve a large period, let's say, one year (to be gentle on the server).

Reply:
ac37cc2 refactored  `download_mrms_data()` to allow downloading the data over a time range. Tretrieve the data from the server. Since this relies on an external server, tests of this function were removed.